src/main/java/core/basesyntax/model/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.model;
+
+public class Rectangle extends Color {
```

Comment:
The class 'Rectangle' should not extend 'Color'. Instead, it should have a color property. Inheritance is not the right way to associate a rectangle with a color because 'Rectangle' is not a type of 'Color', but rather 'Rectangle' has a 'Color'.